### PR TITLE
[Qt] Use setFocusPolicy instead of a custom class

### PR DESCRIPTION
**Pull request by scribam**
Please let me know if what I have done make sense

### CODE REVIEW

File: rpcs3/rpcs3qt/game_list_frame.cpp [hunk 1]
```diff
@@ -146,7 +145,7 @@ game_list_frame::game_list_frame(std::shared_ptr<gui_settings> settings, const R
 
 	gameList = new game_list();
 	gameList->setShowGrid(false);
-	gameList->setItemDelegate(new table_item_delegate(this));
+    gameList->setFocusPolicy(Qt::NoFocus);
```
Comment (danilaml) [inline rpcs3/rpcs3qt/game_list_frame.cpp]: spaces

**Comment by Megamouse** [comment]
This is a completely different behaviour.
First of all, this removes the ability to navigate through the lists by keyboard.
Then it removes the blue Focus indication, so you won't see the item in focus.

As much as I would have liked this to work, there is still a reason for the delegate

**Comment by danilaml** [comment]
Won't something like setting a stylesheet to `QListView { outline: none; }` work?

**Comment by scribam** [comment]
@danilaml I tried your suggestion but it didn't work for me.
Thanks for the review and your feedback, I close this invalid PR.

**Pull request closed without merging**